eclipse-scout-core/test/form/fields/smartfield/SmartFieldSpec.ts

```diff
@@ -387,6 +388,36 @@ describe('SmartField', () => {
       expect(popup._field.tooltip().$container.find('.glasspane').length).toBe(0);
     });
 
+    it('delegates lookup events to original field', () => {
+      let field = createFieldWithLookupCall({
+        touchMode: true
+      });
+      let prepareLookupCallCounter = 0;
+      let lookupCallDoneCounter = 0;
+      let onPrepareLookupCall = () => {
+        prepareLookupCallCounter++;
+      };
+      let onLookupCallDone = () => {
+        lookupCallDoneCounter++;
+      };
+      field.on('prepareLookupCall', onPrepareLookupCall.bind(field));
+      field.on('lookupCallDone', onLookupCallDone.bind(field));
+      field.render();
+      jasmine.clock().tick(500);
+      JQueryTesting.triggerClick(field.$field);
+      jasmine.clock().tick(500);
+
+      let popup = field.popup as SpecSmartFieldTouchPopup;
+      const oldPrepareLookupCallCounter = prepareLookupCallCounter;
```

Comment:
I would prefer "let" here. More because of consistency, than technical correctness.